packages/types/src/studio/studio-server-types.ts

```diff
@@ -1,14 +1,46 @@
 import type { Router } from 'express'
+import type { AxiosInstance } from 'axios'
+
+interface RetryOptions {
+  maxAttempts: number
+  retryDelay?: (attempt: number) => number
+  shouldRetry?: (err?: unknown) => boolean
+  onRetry?: (delay: number, err: unknown) => void
+}
+
+export interface StudioBrowser {
```

Comment:
does a similar type exist elsewhere we can reference here? Then again I think some of those might actually be runtime objects

Reply:
Going to use the Cypress types here.